config/config/sc-config.yaml

```diff
@@ -976,52 +976,53 @@ opensearch:
 
 fluentd:
   enabled: true
```

Comment:
Move to common

Reply:
It'll break the dev flavour :disappointed: